BuildAndTest.cmd

```diff
@@ -57,17 +63,9 @@ echo         public const string Version = AssemblyVersion + Prerelease;
 echo     }                                                                          >> %DRV_VERSION_CONSTANTS%
 echo  }                                                                             >> %DRV_VERSION_CONSTANTS%
 
-if NOT exist "GeneratedKey.snk" (
-sn -k GeneratedKey.snk
-)
-
 @REM Build all code
 %~dp0.nuget\NuGet.exe restore src\Everything.sln -ConfigFile .nuget\NuGet.Config
```

Comment:
You can delete lines 66 & 67. They are already called in the BeforeBuild.cmd script.